docker/stand-alone/docker-compose.yaml

```diff
@@ -0,0 +1,34 @@
+version: '2'
+services:
+  sam-app:
+    image: broadinstitute/sam:stand-alone
```

Comment:
Not crucial, although it would be helpful, and it may not even belong here, to give instructions on how to build and run SAM from local sources. I see in the main README.md that there is the `./docker/build.sh jar -d build`, but tying it all together explicitly on how I build SAM locally then run that local Docker image would be helpful.

I believe I will need to do my own PR to add a new resource type to SAM, so I'm anticipating that. :)

Reply:
Will update with these instructions